modules/services/scrutiny-collector.nix

```
@@ -0,0 +1,127 @@
+{ config, osConfig, lib, pkgs, ... }:
+
+let
+  inherit (lib) mkEnableOption mkIf mkOption mkPackageOption types mdDoc;
+
+  cfg = config.services.scrutiny-collector;
+in {
+  meta.maintainers = [ lib.maintainers.dudeofawesome or "dudeofawesome" ];
```

If you're not in the Nixpkgs maintainer list then you'll need to add yourself to `maintainers.nix` in this repository and change this line to
```suggestion
  meta.maintainers = [ lib.hm.maintainers.dudeofawesome ];
```